sessions/react-data-fetching/star-wars/pages/characters/luke-skywalker.js

```diff
@@ -0,0 +1,15 @@
+import Card from "../../components/Card";
+import Layout from "../../components/Layout";
+
+export default function Character() {
```

Comment:
The transition from the static to the dynamic route might be a bit hard to understand. Maybe we can prepare somthing.

- rename file to `pages/characters/1.js`
- add `id` as a constant that needs toe be replaces with a query param via `useRouter`: `const id = 1;`
- pass and render the  `id` to the `Card` so it does make some sense in the beginning: `<Card id={id} ... />`

Reply:
Good idea, changed it this way!